### PR TITLE
src/pages: add ResultsDetailPage with tabs component

### PR DESCRIPTION
**Pull request by maceksimon**
Adding a results detail page:

* `ResultsDetailPage` to show results for an individual user
* `RoutesTabs` component to switch between different tabs on results page
* new routes for different tabs to `routes_conf`
* translations

### CODE REVIEW

File: src/i18n/en.toml [hunk 1]
```diff
@@ -392,6 +392,9 @@ titleRecentChallenges = "Recent completed challenges"
 titleResults = "Results"
 titleUpcomingChallenges = "Upcoming challenges"
 titleYourResultsSince = "Your results since {date}"
+tabConsistency = "Consistency"
```
Comment (tmszi) [inline src/i18n/en.toml:395]: Pravidelnost -> Frequency

Look at old DPNK app code trans strings please.

Comment (maceksimon) [reply src/i18n/en.toml]: In this context, is it better to use `regularity`?
See https://github.com/auto-mat/do-prace-na-kole/blob/master/apps/dpnk/locale/en/LC_MESSAGES/django.po#L4370

Comment (tmszi) [reply src/i18n/en.toml]: Ok. Yes are right.

**Comment by tmszi** [comment]
detail_report

File: src/components/results/ResultsTabs.vue [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+    // locked tabs - exposed for testing and further logic
+    const lockedTabs = props.locked;
+
+    /**
+     * Checks if the given tab is locked.
+     * @param tab the tab to check
+     */
+    const isLocked = (tab: ResultsTabsOption): boolean => {
+      if (!lockedTabs.length) return false;
```
Comment (tmszi) [inline src/components/results/ResultsTabs.vue:46]: What about [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator)?

Comment (maceksimon) [reply src/components/results/ResultsTabs.vue]: Updated.
Generally, I tend to use early returns for better readability, but in a simple function, I agree this is good.

**Pull request merged**